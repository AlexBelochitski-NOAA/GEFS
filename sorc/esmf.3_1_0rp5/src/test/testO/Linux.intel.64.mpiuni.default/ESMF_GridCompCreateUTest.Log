20190610 112240.939593 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112240.939860 INFO     PET0  Beginning Test, file ESMF_GridCompCreateUTest.F90, line          67
20190610 112240.939877 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112240.941335 INFO     PET0  PASS  Creating a Component Test, ESMF_GridCompCreateUTest.F90, line          80
20190610 112240.941513 INFO     PET0  PASS  Destroying a Component Test, ESMF_GridCompCreateUTest.F90, line          92
20190610 112240.941522 INFO     PET0  Number of failed tests:           0
20190610 112240.941527 INFO     PET0  Ending Test, file ESMF_GridCompCreateUTest.F90, line         218
