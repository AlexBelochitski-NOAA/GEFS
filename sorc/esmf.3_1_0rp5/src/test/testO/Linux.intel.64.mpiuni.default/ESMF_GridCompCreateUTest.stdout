  Beginning Test, file ESMF_GridCompCreateUTest.F90, line          67
  NUMBER_OF_PROCESSORS           1
  PASS  Creating a Component Test, ESMF_GridCompCreateUTest.F90, line          8
 0
  PASS  Destroying a Component Test, ESMF_GridCompCreateUTest.F90, line         
  92
  Number of failed tests:           0
  Ending Test, file ESMF_GridCompCreateUTest.F90, line         218
